CountItems/masterProcess.js

```diff
@@ -15,9 +19,19 @@ const loggerConfig = {
     dump: 'error',
 };
 
+let waitingForPromScraping = false;
+
 werelogs.configure(loggerConfig);
 const log = new werelogs.Logger('S3Utils::CountItems::Master');
 
+const prometheusPollingPeriod = Number.parseInt(process.env.PROMETHEUS_POLLING_PERIOD, 10) > 0
+    ? Number.parseInt(process.env.PROMETHEUS_POLLING_PERIOD, 10)
+    : 30;
+
+const prometheusPollingAttempts = Number.parseInt(process.env.PROMETHEUS_POLLING_ATTEMPTS, 10) > 0
+    ? Number.parseInt(process.env.PROMETHEUS_POLLING_ATTEMPTS, 10)
+    : 5;
```

Comment:
should introduce a function to avoid duplicated parsing (i.e. opportunity for typo):

```suggestion
function tryParseInt(s, defaultValue) {
  const v = Number.parseInt(s, 10);
  return v > 0 ? v : defaultValue;
}

const prometheusPollingPeriod = tryParseInt(process.env.PROMETHEUS_POLLING_PERIOD, 30);
const prometheusPollingAttempts = tryParseInt(process.env.PROMETHEUS_POLLING_ATTEMPTS, 5);
```

Reply:
Indeed , done here : https://github.com/scality/s3utils/pull/312/commits/902a90af79f05379d2e978a489b3df5929c615e5 

_please note that all commits will be squashed once the approvals complete_